src/components/benefit-cta-link.vue

```diff
@@ -63,6 +64,7 @@ export default {
       return link
     },
     onClick: function (link) {
+      this.sendStatistics(this.droits, this.type, this.benefit.id)
```

Comment:
Je pense que c'est une très bonne chose d'utiliser le mixin Resultats pour récupérer `this.droits` et ne pas dupliquer de code mais je ne crois pas que l'on souhaite récupérer cette statistique pour le moment.

Le mixin `StatisticsMixin` utilise notre outil de [stats custom](https://github.com/betagouv/aides-jeunes-stats-recorder) et aujourd'hui si l'on regarde le [schéma mongo](https://github.com/betagouv/aides-jeunes-stats-recorder/blob/master/schemas/benefitsRecords.mongoose.js) on n'accepte que les évènements `show` (page de résultat affichée) et `showDetails` (un utilisateur affiche les détails d'une aide depuis la page de résultat).

Pour le moment on avait décidé de ne pas se préoccuper de si un utilisateur clique sur un lien le redirigeant vers un autre service, mais sachant que maintenant c'est implémenté on devrait discuter de si on intègre cette statistique ou pas

Reply:
J'ai l'impression que l'émission de l'évènement Matomo [se fait ailleurs](https://github.com/betagouv/aides-jeunes/blob/master/src/components/benefit-cta.vue#L6-L10).

Reply:
En effet je sépare dans deux PR différentes